sdk/storage/azure_storage_blob/tests/framework/test_account.rs

```diff
@@ -0,0 +1,53 @@
+use std::sync::{Arc, Once};
+
+use azure_core::{credentials::Secret, TransportOptions, Uuid};
+use azure_core_test::TestContext;
+use azure_data_cosmos::{CosmosClientOptions, Query}; // Get rid of this
+use futures::StreamExt;
+use reqwest::ClientBuilder;
+use time::{macros::format_description, OffsetDateTime};
+
+/// Represents a Azure Storage account for testing purposes.
+///
+/// A [`TestAccount`] serves two main purposes:
+/// * Managing connection information to make it easy to connect to the account the tests are using.
+/// * Provide a unique ID to each test and automatic clean-up of resources.
+pub struct TestAccount {
+    pub context_id: String,
+    endpoint: String,
+    key: Secret,
```

Comment:
DAC?

Reply:
We likely only need `account_name` here for Storage's purposes

Reply:
Why are you even checking in a separate test framework? We have recorded tests - like with every language - coming soon.

Reply:
Thanks for the clarification here Heath! This was heavily cribbed from [cosmosdb](https://github.com/Azure/azure-sdk-for-rust/blob/main/sdk/cosmos/azure_data_cosmos/tests/framework/test_account.rs), but good to know this is not necessary in our context.